spec/pane-spec.coffee

```diff
@@ -375,6 +377,27 @@ describe "Pane", ->
       pane.destroyInactiveItems()
       expect(pane.getItems()).toEqual [item2]
 
+  describe "::showSaveDialog()", ->
+    pane = null
+    item =  null
+
+    beforeEach ->
+      item = new Item("A")
+      pane = new Pane(items: [item], applicationDelegate: atom.applicationDelegate)
+      atom.applicationDelegate.showSaveDialog = jasmine.createSpy("atom.showSaveDialog")
+
+    describe "when the item implements a showSaveDialog method", ->
+      it "calls item.showSaveDialog and saves the item with the results of the returned promise", ->
+        item.showSaveDialog = jasmine.createSpy("item.showSaveDialog")
+        pane.showSaveDialog(item)
+        expect(item.showSaveDialog).toHaveBeenCalled()
+        expect(atom.applicationDelegate.showSaveDialog.callCount).toEqual(0)
+
+    describe "when the item doesn't implement a showSaveDialog method", ->
+      it "calls atom.showSaveDialog and saves the item with the results of the returned promise", ->
```

Comment:
You're describing more behavior than is actually tested here. Should fix one or the other.